patches/ios-web_view-public-cwv_web_view.h.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/ios/web_view/public/cwv_web_view.h b/ios/web_view/public/cwv_web_view.h
+index c17c6eb810f3122705cc0124e68e1b4a32cd10b8..c76bc2193fb3230585e719e35db730d24eee60e1 100644
+--- a/ios/web_view/public/cwv_web_view.h
++++ b/ios/web_view/public/cwv_web_view.h
+@@ -71,7 +71,7 @@ CWV_EXPORT
+ //   (not implemented for CWVWebView)
+ //   |visibleURL| is the bad cert page URL. |lastCommittedURL| is the previous
+ //   page URL.
+-@property(nonatomic, readonly) NSURL* visibleURL;
++@property(nonatomic, readonly, nullable) NSURL* visibleURL;
```

Comment:
why do we want this to be nullable?

Reply:
A couple reasons: 
- Converting GURL to NSURL is not guaranteed to return non-nil because Apple's implementation of NSURL is more lenient, thus `net::NSURLWithGURL` can return nil.
- I noticed that in some cases that `visibleURL` was an invalid GURL which returned nil (see point 1)

With refactors on our URL access we could likely remove this but because of point one I'd rather keep it and always have a chance to check for nil on the Swift side